### PR TITLE
style: replaced .container.is-fluid

### LINKED ISSUE #9245: refactor: container size to tailwind
Replace `container is-*` to tailwind equivalent

- ref #9013 
- https://bulma.io/documentation/layout/container/
- https://tailwindcss.com/docs/container

### PR DESCRIPTION
**Pull request by pbkompasz**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [X] Refactoring

## Context

- [X] Closes #9245
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [X] My contribution builds **clean without any errors or warnings**
- [X] I've merged recent default branch -- **main** and I've no conflicts
- [X] I've tried to respect high code quality standards
- [X] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [X] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=16hqk8Tjugo9AwSzKkxXc1dsRK7iJetpx1YBaoXYwdHzuNAt&usdamount=50&donation=true)

#### Community participation

- [ ] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Review by roiLeo — APPROVED** [review]

**Review by preschian — APPROVED** [review]

File: components/collection/drop/DropContainer.vue [hunk 1]
```diff
@@ -7,7 +7,7 @@
       @model-value="isLoading = false" />
     <CountdownTimer />
     <hr class="text-color my-0" />
-    <div class="container is-fluid">
+    <div class="w-full px-[2.5rem]">
```
Comment (newraina) [inline components/collection/drop/DropContainer.vue:10]: seems `.container.is-fluid` not equal to `.w-full.px-[2.5rem]`:

```css
.container.is-fluid {
    padding-left: 2.5rem;
    padding-right: 2.5rem
}

@media screen and (max-width: 768px) {
    .column-padding {
        margin:0 -12px!important;
        padding-left: 10px!important;
        padding-right: 10px!important
    }

    .container.is-fluid {
        padding-left: 1.25rem;
        padding-right: 1.25rem
    }
}
```

Comment (roiLeo) [reply components/collection/drop/DropContainer.vue:10]: @newraina I've double check https://bulma.io/documentation/layout/container/#fluid-container and it look like it's always `32px`

so maybe instead we should have `w-full px-2`

Comment (roiLeo) [reply components/collection/drop/DropContainer.vue:10]: Nice find, I totally forgot about:
```scss
$fluid-container-padding: 2.5rem;
$fluid-container-padding-mobile: 1.25rem;
```
in that case I would replace `w-full px-[2.5rem]` to `w-full px-5 md:px-10`

File: components/collection/CollectionHeader/CollectionBanner.vue [hunk 2]
```diff
@@ -5,7 +5,8 @@
     <div class="collection-banner-shadow"></div>
 
     <section class="h-full py-8">
-      <div class="container is-fluid collection-banner-content">
+      <!-- <div class="container mx-auto px-[2.5rem] collection-banner-content"> -->
```
Comment (roiLeo) [inline components/collection/CollectionHeader/CollectionBanner.vue:8]: remove if not used

**Review by roiLeo — CHANGES_REQUESTED** [review]
- https://github.com/kodadot/nft-gallery/pull/9279#discussion_r1478259771

**Review by roiLeo — COMMENTED** [review]
please update the remaining 

Capture d’écran 2024-02-07 à 8 40 19 AM

**Review by roiLeo — COMMENTED** [review]
Double check navbar

Screenshot 2024-02-07 at 09-55-08 KodaDot - NFT Market Explorer

**Comment by prury** [comment]
build/tests not running

**Review by roiLeo — APPROVED** [review]

**Comment by roiLeo** [comment]
> build/tests not running

double check

**Comment by prury** [comment]
- mobile: 
[1](https://github.com/kodadot/nft-gallery/assets/36627808/b336e8b9-333a-4ea7-a09b-062933050af5), [2](https://github.com/kodadot/nft-gallery/assets/36627808/bd2b20db-4749-4993-afbd-97b2ccc28254)

- lateral margin changed and also text color from collection title: 
https://github.com/kodadot/nft-gallery/assets/36627808/e985bf8c-a587-4295-a0d4-537b02448d22

- [Alignment](https://github.com/kodadot/nft-gallery/assets/36627808/a276d6e4-d4cf-4336-be33-f89696759231) ,[Alignment](https://github.com/kodadot/nft-gallery/assets/36627808/afd6bac0-be93-4ebf-8d1a-71398c5cdf79)

- General problem: 
https://github.com/kodadot/nft-gallery/assets/36627808/521dc9d1-f02c-4a9f-83f8-3ec52bcd0a86

- Galleryitem
https://github.com/kodadot/nft-gallery/assets/36627808/e7d2599e-58c0-43bb-854c-1bb2573f4660

**Comment by pbkompasz** [comment]
@prury Fixed those, for most of them the issue was `px-5` was not applying `padding: 1.25rem` but some other value.

**Comment by prury** [comment]
image

**Pull request closed without merging**